controlers/contacts.js

```diff
@@ -0,0 +1,35 @@
+const Contact = require("../models/Contact");
+
+const getAllContacts = async () => {
+  try {
+    return await Contact.find();
+  } catch (error) {
+    throw new Error("Error getting contacts from database");
+  }
+};
```

Comment:
```suggestion
const getAllContacts = async ({onwer}) => {
  try {
    return await Contact.find({owner});
  } catch (error) {
    throw new Error("Error getting contacts from database");
  }
};
```
Mam nadzieję ze ma to sens, jak nie to pisz na slaku.